src/ccl/app.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+import { ROUTER_DIRECTIVES } from '@angular/router';
+
+@Component({
+  selector: 'cc-lab',
+  template: '<router-outlet></router-outlet>',
+  directives: [ROUTER_DIRECTIVES]
+})
+export class AppComponent {}
```

Comment:
It's AppComponent because the style convention I've seen in every angular project so far is for the top level component to be called this. LabComponent is now the top-level component for the lab route.